ocr_service/main.py

```
@@ -0,0 +1,55 @@
+)
+logger = logging.getLogger(__name__)
+
+# Debug: Print environment variables to help diagnose issues
+logger.info("==== DEBUG: Environment Variables ====")
+logger.info(f"GEMINI_API_KEY exists: {'Yes' if 'GEMINI_API_KEY' in os.environ else 'No'}")
+logger.info(f"GEMINI_MODEL: {os.environ.get('GEMINI_MODEL', 'Not found')}")
+logger.info(f"DEBUG setting: {os.environ.get('DEBUG', 'Not found')}")
+logger.info("==== End Environment Debug ====")
+
+# Import API endpoints
+from api.endpoints.ocr import router as ocr_router
```

Ruff (0.8.2)

Module level import not at top of file

(E402)